packages/block-library/src/common.scss

```diff
@@ -81,3 +81,34 @@
 .items-justified-space-between {
 	justify-content: space-between;
 }
+
+.screen-reader-text {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	-webkit-clip-path: inset(50%);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+}
+
+.screen-reader-text:focus {
+	background-color: $gray-300;
+	clip: auto !important;
+	clip-path: none;
+	color: #444;
```

Comment:
This color doesn't seem like it would work everywhere. Same with the background color above, no? Can we be smarter here? Or perhaps just simpler? Maybe we use a `white` background color and `black` text and focus style?

Reply:
I just copied the already existing styles from the `VisuallyHidden` component which is the same thing but for the editor. I guess we should be consistent.

Reply:
Can we update the VisuallyHidden CSS then? 🙈 

_I hear you, we can do this separately. I just see an opportunity here._

Reply:
I can update, should I use the CSS from here? https://github.com/WordPress/gutenberg/pull/30141#discussion_r600242830

Reply:
Yes. It's just the white and black colors, and the use of 1em instead of arbitrary pixel values.